services/mailer/mail_repo.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package mailer
+
+import (
+	"bytes"
+	"fmt"
+
+	"code.gitea.io/gitea/models"
+)
+
+// SendRepoTransferNotifyMail triggers a notification e-mail when a pending repository transfer was created
+func SendRepoTransferNotifyMail(doer, newOwner *models.User, repo *models.Repository) error {
+	var (
+		emails      []string
+		destination string
+		content     bytes.Buffer
+	)
+
+	if newOwner.IsOrganization() {
+		users, err := models.GetUsersWhoCanCreateOrgRepo(newOwner.ID)
+		if err != nil {
+			return err
+		}
+
+		for i := range users {
+			emails = append(emails, users[i].Email)
+		}
+		destination = newOwner.DisplayName()
+	} else {
+		emails = []string{newOwner.Email}
+		destination = "you"
+	}
+
+	subject := fmt.Sprintf("%s like to transfer \"%s\" to %s", doer.DisplayName(), repo.FullName(), destination)
```

Comment:
Isn't there a way of localising this?

The message should be:

```suggestion
	subject := fmt.Sprintf("%s would like to transfer \"%s\" to %s", doer.DisplayName(), repo.FullName(), destination)
```

Reply:
> ... a way of localising

not sure I did stick with the eng one as this is most similar with how other email are generated - and one thing we should have in mind to improve ... :/

Reply:
created https://github.com/go-gitea/gitea/issues/14822

Reply:
https://github.com/go-gitea/gitea/blob/3d8b5ad5f37f495e50d83bbad214f8061f8a5ac4/services/mailer/mail.go#L227-L232